docsets/SpigotMC/docset.json

```diff
@@ -1,17 +1,21 @@
 {
     "name": "SpigotMC",
-    "version": "1.12.2",
+    "version": "1.15.1",
     "archive": "SpigotMC.tgz",
```

Comment:
You need to update the archive at this path as well.

Reply:
what do you mean by "update the archive"?  The latest SpigotMC.tgz is in 1.15.1 version folder.

Reply:
I mean the archive at this path: `docsets/SpigotMC/SpigotMC.tgz`. The `specific_version` docset archive and the main docset archive are separate.